examples/calculations/submit_md_using_config.py

```diff
@@ -0,0 +1,31 @@
+"""Example code for submitting single point calculation"""
+
+from aiida.engine import run_get_node
+from aiida.orm import load_code
+from aiida.plugins import CalculationFactory
+
+from aiida_mlip.data.config import JanusConfigfile
+from aiida_mlip.helpers.help_load import load_structure
+
+# And the required inputs for aiida
+metadata = {"options": {"resources": {"num_machines": 1}}}
+code = load_code("janus@localhost")
+
+# Let'say we have a structure already loaded here that we want to use
+structure = load_structure("/home/federica/prova_janus/waterxyz/water.xyz")
+
+# All the other paramenters we want them from the config file
+# We want to pass it as a AiiDA data type for the provenance
+config = JanusConfigfile(
+    "/home/federica/aiida-mlip/tests/calculations/configs/config_janus_md.yaml"
+)
+
+# Define calculation to run
+MDCalculation = CalculationFactory("janus.md")
+
+# Run calculation
+result, node = run_get_node(
+    MDCalculation, code=code, struct=structure, metadata=metadata, config=config
```

Comment:
It might be nice to add (here or the singlepoint example) explicit inputs as well as the config, so it's clear both are possible?

Reply:
Technically, `structure`, `code`, and `metadata` are explicit inputs. `Code` and `metadata` are just for aiida, so that's not helpful but `struct` is for janus (and actually at the moment in that config file there is a different structure used, but thet struct I pass here overrides it)

Reply:
also as a reminder for myself, I need to change the paths in the config and the submission files cause they are path in my localcomputer that I am just using to see if things work